packages/kit/src/exports/node/index.js

```diff
@@ -189,3 +192,22 @@ export async function setResponse(res, response) {
 		}
 	}
 }
+
+/**
+ * TODO
+ * @param {string} file
+ */
+export function readFile(file) {
+	const stats = fs.statSync(file);
```

Comment:
The `statSync` makes me uncomfortable, but I realize that the alternative to that is for each adapter's `readAsset` to be async.

Do we want to do any prod-only caching of asset stats? I'm not entirely certain what sorts of use cases we're trying to enable here, exactly. Do we expect that someone may call `readAsset` many times for the same path? Do we hope that they don't do that?

Reply:
What is it about `statSync` that makes you nervous?

Reply:
It being synchronous and thus blocking any other concurrent requests to the server while we wait for the filesystem.